doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,322 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
+
+1) Agreement secured on the use of trials results data, including:
+a.	Detail level of data required to validate
+b.	Acceptable data values indicating valid results
+c.	Detail level of data that will be published
+
+2) RI-1 Labs are successfully stood up in all target environments
+* Vendor (3rd party)
+*	Community (OPNFV)
+*	LaaS (e.g. UNH)
+
+3) Engaged NVFI vendors successfully configure their infrastructure per RI-1 and run the RC-1 test suite and are able to provide expert feedback
+
+4) Engaged VNF vendors are able to validate that they can instantiate and run rudimentary validation of VNF functionality on more than one conformant NVFI
+
+<a name="9.5.2.7"></a>
+### 9.5.2.7 Initiation 
+
+<a name="9.5.2.7.1"></a>
+#### 9.5.2.7.1 Objectives of RI-1/RC-1 Trials
+
+The object is to quantitively and qualitatively assess and evaluate the following CNTT requirements, methods, and support processes:
+*	RI-1 implementation support methods ( i.e. cookbooks, installation manuals, how to guides etc.)
+* RI-1 Specifications
+* RC-1 Test Suite 
+* TC Traceability
+* Test Pass Criteria
+* Benchmark Data
+ * Other?
+
+Overall, feedback from the trials and issues and gaps found shall be used to enhance and improve the CNTT approach. Enhancements to future releases will/shall be identified accordingly.
+
+<a name="9.5.2.7.2"></a>
+#### 9.5.2.7.2 Trial Participant Interaction with the Community
+
+The focus of the field trials is on the test suites and CNTT methods, not on the systems under test. A process is being developed to identify issues and gaps and managing how they are reported.
+
+CNTT will work very closely with field trial partners (NFVI vendors, VNF vendors, or system integrators) and agree on labs that will be used for the trial. CNTT will take all necessary measures to protect the intellectual property rights (IP rights) for all partners involved in those trials. All Reports and findings will be vetted carefully and only published after being approved by all parties concerned. No test results or records will be kept in any public records without the consent of the participants.
+
+The targeted repositories for this information are:
+
+CNTT GitHub
+
+*  GitHub Code
+  
+*  GitHub Projects
+  
+*  GitHub Issues
+  
+OPNFV 
+
+*  (Where?)
+
+<a name="9.5.2.7.3"></a>
+#### 9.5.2.7.3 Test Case Identification
+
+Specific test cases for the field trials will be documented and provided to the participants based upon the CNTT RI-1  and RC-1 work streams requirements. The technical testing methods, procedures and documentation shall be provided by these work streams. 
+
+<a name="9.5.2.7.4"></a>
+#### 9.5.2.7.4 Vendor Solicitation/Commitment
+
+NFVI and VNF vendor members will be solicited for participation in the trials. The vendors will be required to commit fully to the assessment and evaluation  processes. As previously mentioned, additional discussion is needed to define what results data and at what level is acceptable to be shared.
```

Comment:
Why not all vendors -- NFVI, VNF, OpenStack, LaaS, Testing service providers, etc.

Reply:
resolved